l10n_br_account_product/views/res_country_view.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_l10n_br_country_state_icms_table_form" model="ir.ui.view">
+            <field name="name">res.country.state</field>
+            <field name="model">res.country.state</field>
+            <field name="inherit_id" ref="l10n_br_base.view_l10n_br_country_state_form"/>
+            <field name="arch" type="xml">
+                <field name="ibge_code" position="after">
+                    <group colspan="4">
+                    <notebook>
+                        <page string="Impostos">
+                            <separator colspan="4" string="Impostos sobre Produtos" />
+                            <field colspan="2" nolabel="1" widget="one2many_list" name="product_tax_definition_line"/>
+                        </page>
+                    </notebook>
+                    </group>
+                </field>
+            </field>
+        </record>
+
+               <record id="l10n_br_tax_definition_company_product_form" model="ir.ui.view">
```

Comment:
@renatonlima , need to fix the indenting and change the id to something like this "l10n_br_tax_definition_state_product_form" to avoid error with other vision